README.md

```diff
@@ -56,7 +56,7 @@ Gradle can be used to build projects developed in various programming languages.
 - Edit its `build.gradle` file adding this:
 ```
 plugins {
-  id 'org.sonatype.gradle.plugins.scan' version '2.0.10' // Update the version as needed
+  id 'org.sonatype.gradle.plugins.scan' version '2.1.1' // Update the version as needed
```

Comment:
We're updating the minor version because we're adding a backwards compatible feature right? Should the patch version be reset to 0? (or was there also some bug fixes?)